javascript.js

```diff
@@ -63,8 +63,22 @@ const switchButton = document.querySelector('.button__bookmark')
 // })
 
 /* Switch bookmark Button per if..else method) */
-switchButton.addEventListener('click', () => {
-  if (switchButton.classList.contains('button__bookmark--clicked')) {
-    switchButton.classList.remove('button__bookmark--clicked')
-  } else switchButton.classList.add('button__bookmark--clicked')
+// switchButton.addEventListener('click', () => {
+//   if (switchButton.classList.contains('button__bookmark--clicked')) {
+//     switchButton.classList.remove('button__bookmark--clicked')
+//   } else switchButton.classList.add('button__bookmark--clicked')
+// })
+
+/* Switch bookmark Button für alle bookmarks */
```

Comment:
Please remove code which is commented out! 🙂 
Also, I LOVE your PullRequest description!

Reply:
Thanks! 🙂
And I will remove the mentioned code :wink: